lib/src/lifecycle_module.dart

```diff
@@ -52,6 +53,8 @@ enum LifecycleState {
 /// Intended to be extended by most base module classes in order to provide a
 /// unified lifecycle API.
 abstract class LifecycleModule extends SimpleModule with Disposable {
+  @visibleForTesting
```

Comment:
We probably don't even need this, someone might want to enable it this way in an app. Maybe make it private and module-level and then add a static or module-level function to flip it to true.

So, at the module level (or statically on the `LifecycleModule` class):

```dart
bool _verboseLogging = false;

void enableVerboseLogging() => _verboseLogging = true;
```

Reply:
Maybe both!

Reply:
I'll leave it per-object for now.

Reply:
I'm probably missing something subtle, but I'm not sure why the property would need to be private with a function to set it.